### PR TITLE
Col 20

### PR DESCRIPTION
**Pull request by WiktoriaKrzyzanska**
Added some aesthetic to make it look prettier. Check out Enter :)

### CODE REVIEW

File: core/src/model/views/Lobby.java [hunk 1]
```diff
@@ -54,9 +57,22 @@ public Lobby(final MoneyLandGame game){
         parent.serverHandler.setupConnectWithLobbyScreen(this);
         String playerNickForServer = new String(parent.getPlayerNick());
         parent.serverHandler.sendMessage(playerNickForServer); //send message to server
-
+        camera = new OrthographicCamera();
```
Comment (Ikar17) [inline core/src/model/views/Lobby.java:66]: Is it necessary? You can use camera from MoneyLandGame (parent.camera)

**Review by krzysztof2430 — APPROVED** [review]
Good job

**Review by Ikar17 — APPROVED** [review]
Looks great

**Review by brzezin0 — APPROVED** [review]
10/10

**Pull request merged**